src/translations/de.ts

```diff
@@ -1463,6 +1463,7 @@ export default {
 		"yourCalendars_label": "Deine Kalender",
 		"yourFolders_action": "DEINE ORDNER",
 		"yourMessage_label": "Deine Nachricht",
-		"you_label": "Du"
+		"you_label": "Du",
+		"invalidPastedRecipients_msg": "Die folgende E-Mail-Adressen sind ungültig:"
```

Comment:
Either "die folgenden E-Mail-Adressen" for plural or "die folgende E-Mail-Adresse" for singular (same in de_sie.ts).